phpstan-v67-baseline.neon

```diff
@@ -0,0 +1,96 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Method \"getOrders\" of class \"Shopware\\\\Storefront\\\\Page\\\\Account\\\\Order\\\\AccountOrderPage\" is marked as deprecated, but does not call \"Feature\\:\\:triggerDeprecationOrThrow\"\\. All deprecated methods need to trigger a deprecation warning\\.$#"
```

Comment:
Hi @aragon999, am pasting here comments from the internal core team review:

We should adding those triggerDeprecationOrThrow methods.

Reply:
@dfrancos-hub @ssltg does it really make sense to throw deprecation errors here? :thinking:

Reply:
Yes, because we want to keep the phpstan exceptions as low as possible. I know in the current state it is technically not possible that the deprecation is triggered in these methods, but to trigger the deprecation errors helps us to make sure this will stay until 6.7. As many developers are working on our codebase, this is the best way to ensure compatibility even through automatic merge conflict resolves.